src/main/app/src/app/shared/material-form-builder/form-components/medewerker-groep/medewerker-groep.component.ts

```diff
@@ -199,6 +210,12 @@ export class MedewerkerGroepComponent
         max: this.data.maxGroupNameLength,
       });
     }
-    return "This field has a error";
+    return "This field has an error";
```

Comment:
This is a strange return value.... It doesn't give enough context for a proper "error" message, nor does it fail the code, I don't get when this happens and what the reason is for this result. What can we do with this?

Reply:
Yeah, since the error is not about 1 of the 4 supported possible errors it is apparently not clear what can be the exact error. I can't do much about this @bas-info-nl

Reply:
I did create a translation for it; the least I could do 😉